src/test/resources/mockdata/compositeLines/0009662b-8b80-4001-b704-ca10971f175d.json

```diff
@@ -40,7 +40,8 @@
     {
       "code": "STATE-SUBN",
       "fundId": "e54b1f4d-7d05-4b1a-9368-3c36b75d8ac6",
-      "percentage": 100.0,
+      "distributionType": "percentage",
```

Comment:
Can some of the mockdata have `"distributionType": "amount"`? It might be helpful to cover both cases since now `distributionType` can be `amount` or `percentage`.